modules/standard/Math.chpl

```diff
@@ -902,7 +902,7 @@ module Math {
 
      :rtype: The type of `x`.
    */
-  inline proc max(x, y) {
+  inline proc max(x, y) where !isArray(x) && !isArray(y) {
```

Comment:
should it be something more like, conceptually, "where !isIterable(x)" ? At least including domains, which are also subject to promotion? Or, should this function be "last resort"?

Reply:
Works for me. Seems like something to add to the PR message, then.

Reply:
Good thought, done.